packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -722,7 +722,7 @@ export class RunningSummarizer implements IDisposable {
 		// Set to undefined first, so that subsequent enqueue attempt while summarize will occur later.
 		this.enqueuedSummary = undefined;
 		this.trySummarizeOnce(
-			{ reason: `enqueuedSummary/${reason}` },
```

Comment:
The second change from #16828 is here - This reason string is not updated anymore.